docs/topics/images.md

```diff
@@ -414,6 +414,41 @@ Note that this will have no effect on PNG or GIF files. If you want all images t
 {% image page.photo width-400 format-webp webpquality-50 %}
 ```
 
+### Custom image filter
+
+In `wagtail_hooks.py` register your custom image filter.
+
+This example assumes Willow in combination with Pillow. If you use another image library, or like to support multiple image libraries, you need to update the code inside the run method. See the [Willow documentation](https://willow.readthedocs.io/en/latest/index.html) for more information.
```

Comment:
```suggestion
### Custom image filter

You can add additional image filters that aren't included in Wagtail core. To do so, add a `"register_image_operations"` hook to your `wagtail_hooks.py` file.

This assumes Willow in combination with Pillow. If you use another image library, or like to support multiple image libraries, you need to update the code inside the run method. See the [Willow documentation](https://willow.readthedocs.io/en/latest/index.html) for more information.
```

Reply:
I reworded a bit more, to give some additional background. Hope you like it.